cmake/onnxruntime_fuzz_test.cmake

```diff
@@ -52,13 +53,13 @@ if (onnxruntime_FUZZ_ENABLED)
 	target_include_directories(onnxruntime_security_fuzz PRIVATE ${INCLUDE_FILES})
 	
 	# add link libraries the project
-	target_link_libraries(onnxruntime_security_fuzz libprotobuf onnx_proto onnxruntime protobuf-mutator)
+	target_link_libraries(onnxruntime_security_fuzz ${PROTOBUF_LIB} onnx_proto onnxruntime protobuf-mutator)
```

Comment:
Here ${PROTOBUF_LIB} should be at the right-most. See: https://github.com/microsoft/onnxruntime/blob/main/docs/cmake_guideline.md#static-library-order-matters